package.json

```
@@ -25,7 +25,7 @@
     "ember-cli-babel": "^6.6.0",
     "ember-get-config": "^0.2.4",
     "ember-fetch": "^4.0.0 || ^5.0.0 || ^6.0.0",
-    "ember-simple-auth": "^1.6.0"
+    "ember-simple-auth": "^1.6.0 || ^2.1.0"
```

I think we should also add support for `ember-simple-auth@^3.0.0`:

```suggestion
    "ember-simple-auth": "^1.6.0 || ^2.1.0 || ^3.0.0"
```

---

I somewhat disagree, I believe there should be at least one release with support for node 8 and ember-simple-auth v2 for those that haven't got the latest version or are unable to.

---

I somewhat disagree, I believe there should be at least one release with support for node 8 and ember-simple-auth v2 for those that haven't got the latest version or are unable to.

---

I have added support for ember-simple-auth v3 but not done it in such a way that node v10 is required. In fact, I have also reverted a previous change so that node v6 is also still supported.